examples/kcs-example/src/app/kcs.nft.example.ts

```diff
@@ -30,32 +39,112 @@ export async function kcsNftExample() {
     '0x45871ED5F15203C0ce791eFE5f4B5044833aE10e',
   )
 
-  // TODO bug in openapi - chain and feeCurrency
-  // const minted = await kcsSDK.nft.mintNFT({
-  //   chain: Currency.KCS,
-  //   tokenId: "100000",
-  //   to: "0x687422eEA2cB73B5d3e242bA5456b782919AFc85",
-  //   erc20: "0x687422eEA2cB73B5d3e242bA5456b782919AFc85",
-  //   contractAddress: "0x687422eEA2cB73B5d3e242bA5456b782919AFc85",
-  //   url: "https://my_token_data.com",
-  //   authorAddresses: [
-  //     "0x687422eEA2cB73B5d3e242bA5456b782919AFc85"
-  //   ],
-  //   provenance: true,
-  //   cashbackValues: [
-  //     "0.5"
-  //   ],
-  //   fixedValues: [
-  //     "0.5"
-  //   ],
-  //   fromPrivateKey: "0x05e150c73f1920ec14caa1e0b6aa09940899678051a78542840c2668ce5080c2",
-  //   nonce: 0,
-  //   feeCurrency: Currency.KCS
-  // })
-
-  await kcsSDK.nft.prepareAddNFTMinterAbstraction(
+  const nftAccountBalance = await kcsSDK.nft.getNFTAccountBalance(
     Currency.KCS,
-    '0x687422eEA2cB73B5d3e242bA5456b782919AFc85',
-    '1000',
+    '0x3223AEB8404C7525FcAA6C512f91e287AE9FfE7B',
+    '0x94Ce79B9F001E25BBEbE7C01998A78F7B27D1326',
   )
+
+  const mintedHash = await kcsSDK.nft.mintNFT({
+    chain: MintNft.chain.KCS,
+    tokenId: '100000',
+    to: '0x687422eEA2cB73B5d3e242bA5456b782919AFc85',
+    erc20: '0x687422eEA2cB73B5d3e242bA5456b782919AFc85',
+    contractAddress: '0x687422eEA2cB73B5d3e242bA5456b782919AFc85',
+    url: 'https://my_token_data.com',
+    authorAddresses: ['0x687422eEA2cB73B5d3e242bA5456b782919AFc85'],
+    provenance: true,
+    cashbackValues: ['0.5'],
+    fixedValues: ['0.5'],
+    fromPrivateKey: '0x05e150c73f1920ec14caa1e0b6aa09940899678051a78542840c2668ce5080c2',
+    nonce: 0,
+  })
+
+  const deployHash = await kcsSDK.nft.deployNFTSmartContract({
+    // TODO openapi bug
```

Comment:
Are there still some openapi bugs? I have regenerated code from new version of openapi.

Reply:
probably not, I have to update it as well

Reply:
It should be in master